components/docusign/actions/create-signature-request/create-signature-request.mjs

```diff
@@ -25,6 +25,7 @@ export default {
           account: c.account,
         }),
       ],
+      reloadProps: true,
```

Comment:
Shouldn't this be changed in `components/docusign_developer/actions/create-signature-request/create-signature-request.mjs` as well?